src/lib/server/api/discord.ts

```diff
@@ -160,8 +199,20 @@ export async function logApprovedConfessionViaHttp(
     authorId: Snowflake,
     label: string,
     description: string,
+    attachment: Attachment | null,
     botToken = DISCORD_BOT_TOKEN,
 ) {
+    const fields = [
+        {
+            name: 'Authored by',
+            value: `||<@${authorId}>||`,
+            inline: true,
+        },
+    ];
+    if (attachment) {
```

Comment:
Same concern as in a similar review comment on explicit conditions.
```suggestion
    if (attachment !== null) {
```